src/app/project/template.html

```diff
@@ -22,7 +22,9 @@
       <div fxFlex></div>
       <mat-slide-toggle *ngIf="isAdmin"
                         class="all-projects-toggle"
+                        [disabled]="isProjectsLoading"
                         labelPosition="before"
+                        [matTooltip]="isProjectsLoading ? 'Please wait while projects are loading' : null"
```

Comment:
There is something wrong with logic in parent! it appears occasionally.

this section appears on ` *ngIf="!isInitializing ` but it should we need to show when everytime API call is triggered `isProjectsLoading`

https://user-images.githubusercontent.com/17727069/192952980-9a7af28f-3a77-4748-abcd-cb4ee66ac348.mp4

Reply:
You can ignore, this comment i think the issue was when i was trying to click on the "Show all projects" and cursor was still on that toggle so **tooltip** was not appearing but if you move your cursor from toggle and than hover again it is showing tooltip

Reply:
cc: @Waseem826